packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -582,12 +583,14 @@ class DatePicker extends React.Component {
     }
   }
 
-  setPreSelection = (date) => {
+  setPreSelection = (date,type,value) => {
     const isValidDateSelection = date ? isDayInRange(date, this.props.minDate, this.props.maxDate) : true
     if (isValidDateSelection) {
       this.setState({
         preSelection: date
       })
+      type === "month" ? this.updateAriaLiveStatus(getMonthFromDate(date, this.props)) :
+      type === "year" ? this.updateAriaLiveStatus(value, this.props) :
       this.updateAriaLiveStatus(getLocalizedDateForScreenReader(date, this.props));
```

Comment:
how is this not overwriting the previous lines (even though SR announces only month/year when changed). Can this line be re-checked once

Reply:
No it is not overwriting the previous lines as this is similar to this in if & else condition _**if (type === 'month') {
    this.updateAriaLiveStatus(getMonthFromDate(date, this.props));
} else if (type === 'year') {
    this.updateAriaLiveStatus(value, this.props);
} else {
    this.updateAriaLiveStatus(getLocalizedDateForScreenReader(date, this.props));**_
}